src/components/index.js

```diff
@@ -1,4 +1,4 @@
-export { default as TopNav } from './top_nav/TopNav';
+export { default as Navbar } from './navbar/Navbar';
```

Comment:
I do like the renaming, but are we 100% sure it's been renamed throughout the repo?